observers/slack_resource.cpp

```diff
@@ -0,0 +1,82 @@
+#include <memory>
+#include <vector>
+
+#include "glog/logging.h"
+
+#include "mesos/mesos.hpp"
+
+#include "observers/slack_resource.hpp"
+
+#include "serenity/metrics_helpers.hpp"
+
+namespace mesos {
+namespace serenity {
+
+Try<Nothing> SlackResourceObserver::consume(const ResourceUsage& usage) {
+  std::unique_ptr<ExecutorSet> newSamples(new ExecutorSet());
```

Comment:
Just a question, but what's the convention in this project for using-declarations?  Can we decide on a single way and apply it consistently?  Mesos code style seems to always supply the declarations.